.github/workflows/software.yml

```diff
@@ -0,0 +1,41 @@
+name: build software
+on:
+  schedule:
+    - cron: "0 0 * * 1"
+  push:
+    branches:
+      - master
+    paths:
+      - 'software/**'
```

Comment:
Yolks are the generic fit-it-all image category. Please change it to use that.

Reply:
Has been moved to a dedicated cassandra directory. 
The workflow has been adjusted and now works like all the other ones (like java or erlang) and pushes to yolks instead of software.

Reply:
Github deployed the review without me actually finishing it. That's what I was going to suggest. The same format makes it easy for others to follow and modify.